demo/demo.md

```diff
@@ -0,0 +1,117 @@
+<!--
+The demo.md file is a compiled document. No edits should be made directly to this file.
+README.md is created by running `npm run build:docs`.
+This file is generated based on a template fetched from `./docs/partials/demo.md`
+-->
+
+# Pane
+
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./description.md) -->
+<!-- The below content is automatically added from ./description.md -->
+`<auro-pane>` is a [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) displaying selectable shoulder dates and associated prices.
+<!-- AURO-GENERATED-CONTENT:END -->
+
+## auro-pane use cases
+
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./useCases.md) -->
+<!-- The below content is automatically added from ./useCases.md -->
+The `<auro-pane>` element should be used in situations where users may:
+
+* Select a date to see available flight options
+* See multiple dates with price and availability information
+<!-- AURO-GENERATED-CONTENT:END -->
+
+## Example(s)
+
+## Basic Example
```

Comment:
```suggestion
## Display date

Default using` date` attribute
```

Reply:
The requested change would be a diversion from all recent components docs where the main demo page lists the "Basic" example as the first example.

The attributes are all explained and called out on the API page. We should not divert from this documentation pattern.